src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -230,6 +277,7 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
         }
 
         // Check if the motion between the nearest state and the state to add is valid
+        ++collisionChecks_;
```

Comment:
Is this really the number of calls to the state validity checker? If not, does each collision check take the same amount of time? I'm curious what the intended use is of this variable.